src/Orchard.Web/Modules/Orchard.Dashboards/Web.config

```diff
@@ -7,7 +7,7 @@
         </sectionGroup>
     </configSections>
     <system.web.webPages.razor>
-        <host factoryType="System.Web.Mvc.MvcWebRazorHostFactory, System.Web.Mvc, Version=5.2.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35" />
+        <host factoryType="System.Web.Mvc.MvcWebRazorHostFactory, System.Web.Mvc, Version=5.2.3, Culture=neutral, PublicKeyToken=31bf3856ad364e35" />
```

Comment:
@sebastienros I think that was also discovered when there was a problem with running the site with IIS Expess, but I don't remember now where we got with that.